implementation.js

```diff
@@ -161,7 +159,7 @@ module.exports = function from(arrayLike) {
 		items = arrayFromIterable;
 		len = arrayFromIterable.length;
 	} else if (isString(items)) {
-		return strMatch.call(items, /[\uD800-\uDBFF][\uDC00-\uDFFF]?|[^\uD800-\uDFFF]|./g) || [];
+		return ''.match.call(items, /[\uD800-\uDBFF][\uDC00-\uDFFF]?|[^\uD800-\uDFFF]|./g) || [];
```

Comment:
Here you are creating a new empty string, traversing the prototype for `match` and after executing it you are throwing away the empty string, I think the previous approach of extracting and caching the `match` method is the better option to take.